src/renderer/components/playlist-info/playlist-info.js

```diff
@@ -318,10 +336,27 @@ export default defineComponent({
       this.showDeletePlaylistPrompt = false
     },
 
+    enableQuickBookmarkForThisPlaylist() {
+      if (this.quickBookmarkEnabled) {
+        showToast(this.$t('User Playlists.SinglePlaylistView.Toast["Another playlist is already used for quick bookmark. Click here to use this playlist anyway"]'), 5000, () => {
```

Comment:
**issue (blocking):** Not a fan of this UX. Two reasons:
1. Content gives implication that there could be multiple quick bookmarks active at once.
2. Toast as a makeshift confirmation modal is clunky / unintuitive.

**suggestion:** Make this button just set the playlist as the new quick bookmark. Change content to something like 
> Change quick bookmark from "${playlistName}" to "${playlistName}".

Make the toast:
> "Changed quick bookmark from "${playlistName}" to "${playlistName}". Click to undo.